src/hotspot/share/prims/jvmtiThreadState.hpp

```diff
@@ -231,6 +231,8 @@ class JvmtiThreadState : public CHeapObj<mtInternal> {
   inline JvmtiEnvThreadState* head_env_thread_state();
   inline void set_head_env_thread_state(JvmtiEnvThreadState* ets);
 
+  static bool _seen_interp_only_mode; // needed for optimization
```

Comment:
Say what the flag represents, not why we have it.

Reply:
Thank you for looking at this PR!
Okay, thanks. Will do.

Reply:
Fixed.